### PR TITLE
Doc for custom snippets

### PR DESCRIPTION
**Pull request by qkzk**
This will add some documentation for custom snippets.

### CODE REVIEW

**Comment by qkzk** [comment]
Wrote this since I lost a few hours yesterday trying to figure it.
It's quite simple and useful, I believe it should be documented properly.

**Comment by qkzk** [comment]
The file should be displayed [here](https://www.lunarvim.org/configuration/) if I understand it correctly.

---

Added a description about different kind of snippets (from vs code AKA json) or directly in lua.

File: docs/configuration/08-custom-snippets.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Description
+
+You can add your own snippets to LunarVim.
+
+Custom snippets can be defined as json or lua.
+
+### json version
+
+First create a folder : `~/.config/snippets/` next to your `config.lua`
```
Comment (cpea2506) [inline docs/configuration/08-custom-snippets.md:11]: I see that this path must be `~/.config/lvim/snippets/`

**Comment by qkzk** [comment]
I've PR a fix

**Pull request merged**